Apps/Database/src/Delegates/IEmailDelegate.cs

```diff
@@ -37,7 +37,7 @@ public interface IEmailDelegate
         /// </summary>
         /// <param name="emailId">The Email ID to retrieve.</param>
         /// <returns>The found Email object.</returns>
-        Email GetNewEmail(Guid emailId);
+        Email? GetNewEmail(Guid emailId);
```

Comment:
if null is a valid return, lets add that to the documentation

Reply:
We haven't done that for any other interface so I'd like to stay consistent.

Reply:
sure, we can change that in bulk